src/components/dataset/schemaCreation/DatasetSchemaBuilderView.tsx

```diff
@@ -0,0 +1,780 @@
+                <IconButton
+                  id="datasetSchema-up"
+                  size="small"
+                  color="primary"
+                  className={classes.iconButton}
+                  disabled={selectedTable === -1}
+                  onClick={() => {
+                    const schemaCopy = _.cloneDeep(datasetSchema);
+                    if (selectedColumn > 0) {
+                      swapArrayLocs(
+                        schemaCopy.tables[selectedTable].columns,
+                        selectedColumn,
+                        selectedColumn - 1,
+                      );
+                      setSelectedColumn(selectedColumn - 1);
+                    } else if (selectedColumn === -1 && selectedTable > 0) {
+                      swapArrayLocs(schemaCopy.tables, selectedTable, selectedTable - 1);
+                      setSelectedTable(selectedTable - 1);
+                    }
+                    setDatasetSchema(schemaCopy);
+                  }}
+                >
+                  <i className="fa fa-angle-up" />
+                </IconButton>
+
+                <IconButton
+                  id="datasetSchema-down"
+                  size="small"
+                  color="primary"
+                  className={classes.iconButton}
+                  disabled={selectedTable === -1}
+                  onClick={() => {
+                    const schemaCopy = _.cloneDeep(datasetSchema);
+                    if (
+                      selectedColumn !== -1 &&
+                      selectedColumn < schemaCopy.tables[selectedTable].columns.length - 1
+                    ) {
+                      swapArrayLocs(
+                        schemaCopy.tables[selectedTable].columns,
+                        selectedColumn,
+                        selectedColumn + 1,
+                      );
+                      setSelectedColumn(selectedColumn + 1);
+                    } else if (
+                      selectedTable !== -1 &&
+                      selectedTable < schemaCopy.tables.length - 1
+                    ) {
+                      swapArrayLocs(schemaCopy.tables, selectedTable, selectedTable + 1);
+                      setSelectedTable(selectedTable + 1);
+                    }
+                    setDatasetSchema(schemaCopy);
+                  }}
+                >
+                  <i className="fa fa-angle-down" />
+                </IconButton>
+
+                <IconButton
+                  id="datasetSchema-linkRel"
+                  size="small"
+                  color="primary"
+                  className={classes.iconButton}
+                  style={{ marginLeft: 50 }}
+                  disabled={!datasetSchema.tables || datasetSchema.tables.length < 2}
+                  onClick={() => {
+                    setRelationshipModalDefaultValues({
+                      from: '',
+                      to: '',
+                      expandedTables: {},
+                    });
```

Comment:
This matches the original default value - can we put this into a constant and share it between the two?